azerothcore/azerothcore-wotlk
# Core/Commands: Add new spy command

In the context of in-game player management, sometimes it's useful to know what abusive players are doing.

##### CHANGES PROPOSED:

-  Add a new spy command for game masters and administrators to receive all conversations of followed players.

##### TESTS PERFORMED:

- Build confirmed
- In-game use confirmed in my own server (production)
- Compilation done under Windows and Linux. In-game check on a Linux server.


##### HOW TO TEST THE CHANGES:

- Use command `.spy follow <characterName>` with a GM account
- Make that player whisper another player
- You (GM) should receive the message sent
- Use command `.spy status`. You should get a list with the test subject's name in it.
- Use command `.spy unfollow <characterName>`
- You should not receive any more messages sent or received by that player


##### KNOWN ISSUES AND TODO LIST:

None as for now


##### Target branch(es):

Master
